src/lib.rs

```diff
@@ -38,44 +34,46 @@ struct Component {
     pub cron_expression: String,
 }
 
+#[derive(Args)]
+pub struct CliArgs {}
+
```

Comment:
I hope there is a better way to do this?

Reply:
`spin_trigger::cli::NoCliArgs`